ql/src/java/org/apache/hadoop/hive/ql/metadata/Hive.java

```diff
@@ -3949,7 +3949,7 @@ private static void moveAcidFiles(String deltaFileType, PathFilter pathFilter, F
    */
   protected void replaceFiles(Path tablePath, Path srcf, Path destf, Path oldPath, HiveConf conf,
           boolean isSrcLocal, boolean purge, List<Path> newFiles, PathFilter deletePathFilter,
-          boolean isMmTableOverwrite) throws HiveException {
+          boolean isMmTableOverwrite, boolean isTempTable) throws HiveException {
```

Comment:
Instead of naming the input variable specific for temp table, shall use isNeedRecycle.

Reply:
will change it